new-content/events/summer-school-2021.yaml

```
@@ -12,7 +12,9 @@ header:
     - 'Speakers will be announced soon!'
   cta:
     label: 'Learn more'
-    url: 'https://qiskit.org'
+    url: 'http://qisk.it/qgss2021blog_website_source'
+    segment:
+      action: 'events-summer-school > header > qiskit-org-home'
```

If you change the link, you should also change the segment action
```suggestion
      action: 'events-summer-school > header > ibm-research-blog'
```